lib/commands/serve.js

```diff
@@ -18,7 +18,8 @@ module.exports = Command.extend({
     { name: 'insecure-proxy', type: Boolean, default: false, description: 'Set false to proxy self-signed SSL certificates', aliases: ['inspr'] },
     { name: 'watcher',  type: String, default: 'events', aliases: ['w'] },
     { name: 'live-reload',  type: Boolean, default: true, aliases: ['lr'] },
-    { name: 'live-reload-port', type: Number, description: '(Defaults to port number + 31529)', aliases: ['lrp']},
+    { name: 'live-reload-host', default: '0.0.0.0', type: String, description: 'Defaults to host', aliases: ['lrh'] },
```

Comment:
Maybe i don't get how the command parsing works, but I wanted it to work like this.

```bash
ember serve --host=example.org
```
and then `liveReloadHost` would have the value of `example.org`. It seems like now, it would have `0.0.0.0`